Packages/PackageSources/meridian-simcache/SimObjects/Misc/SimCache/model/SimCache.xml

```diff
@@ -1,9 +1,10 @@
-<?xml version="1.0" encoding="utf-8" ?>
-
-<ModelInfo>
-
+<?xml version="1.0" encoding="utf-8"?>
+<ModelInfo guid="{ffe5f233-1e1b-4e37-a7af-f2a3d0d18b1b}" version="1.1">
```

Comment:
Worth noting, the guid and version attributes in ModelInfo aren't required per the documentation. A GUID is only required for "static scenery objects" with "no animations or interactions". Can always review this when we have a proper version of the SimObject again.